search by title updated

sssilvia9977 opened the pull request.

sssilvia9977 committed:
search by title updated

serban-petrescu commented:
## Grading

Note that you must present at the laboratory one operation out of each feature (to prove that it works) and answer any questions from me related to your code (during the lab):
 1. Create a question and then list all of them.
 
If you are unable to show that the assignment works or answer my questions, this assignment will not be considered as being handed-in.

### Minimum requirements for a passing grade:
 * [X] Browser-based U.I.,
 * [X] Feature 1,
 * [X] Clear MV* separation,
 * [X] Observable model.

### Additional requirements
 * [X] Primitive model-based routing (6)
 * [ ] Feature 2 (8)
 * [ ] Feature 3 (10)

### Bonus features
 * [ ] Bonus feature 1
 * [ ] Bonus feature 2
  
### Bonus requirements
 * [X] Prepare and present mockups (+1.0)
 * [ ] Use Redux for holding the state (+2.0)
 * [X] Use React Router for handling the routing (+1.0)
 * [X] Use a CSS library for styling (+1.0)
 * [ ] Write component tests for your views (+1.0)

**Final grade: 9.** 

### Notes

~~minus 0.5: Search by title seems to be not really finished.~~

sssilvia9977 committed:
assig 2 pentru 3

i have an error when loading assig 2 at the assig 3 rep so i have put it in assig 2 rep
